packages/compass-collection-stats/src/components/collection-stats-item/collection-stats-item.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { css } from '@mongodb-js/compass-components';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const collectionStatsItemStyles = css({
+  marginRight: '12px',
+});
+
+const PrimaryLabelStyles = css({
+  textTransform: 'uppercase',
+  fontSize: '10px',
+  fontWeight: 'bold',
+  display: 'inline-block',
+  marginRight: ' 5px',
+  color: uiColors.gray.dark3,
+});
+
+const LabelStyles = css({
+  textTransform: 'uppercase',
+  fontSize: '10px',
+  fontWeight: 'bold',
+  color: uiColors.gray.base,
+  lineHeight: '12px',
+});
+
+const PrimaryValueStyles = css({
+  color: uiColors.gray.dark3,
+  display: 'inline-block',
+  fontSize: '24px',
+  lineHeight: '24px',
+});
+
+const ValueStyles = css({
+  color: uiColors.gray.base,
+  fontSize: '14px',
```

Comment:
Same with here, does it make sense to replace this with a `Description` or `Body`?
https://www.mongodb.design/component/typography/example/